Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -186,6 +186,10 @@ void ViewTransition::setupViewTransition()
     if (!m_document)
         return;
 
+    ASSERT(m_phase == ViewTransitionPhase::PendingCapture || m_phase == ViewTransitionPhase::Done);
```

Comment:
It's unclear to me how `setupViewTransition()` can be called during the `Done` phase

Reply:
You're right. Fixed.